modules/nodes-pods-understanding-requests-limits.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-using.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nodes-pods-understanding-requests-limits_{context}"]
+= Understanding resource requests and limits
+
+You can specify CPU and memory requests and limits for pods in pod specs, as shown in "Example pod configurations", or the specification of a pod's controlling object.   
+
+CPU and memory _requests_ specify the minimum amount of a resource that a pod needs to run, helping to schedule pods on nodes with sufficient resources. 
+
+CPU and memory _limits_ define the maximum amount of memory a pod can consume, preventing it from consuming excessive memory and potentially impacting other pods on the same node. 
```

Comment:
nit:

> CPU and memory _limits_ define the maximum amount of memory a pod...

You probably want something more generic like "amount of a resource" instead of the memory-specific "amount of memory" there.  Also later in this sentence where you have "excessive memory" you probably want "excessive resources".  Or split into two sentences, one talking about CPU limits and another talking about memory limits?